shared/helpers.js

```diff
@@ -70,3 +71,15 @@ export function translate(format, bindings = {}) {
 
     return ret;
 }
+
+export function currencyExists(currency) {
```

Comment:
Can you rename this function `isKnownCurrency`? Even if it is not known, it doesn't mean it doesn't exist :)